controllers/constants.go

```diff
@@ -64,6 +64,11 @@ const (
 	labelInstanceKey        = "cdap.instance"
 	labelContainerKeyPrefix = "cdap.container."
 
+	// kubernetes security context
+	defaultSecurityContextUID   = 1000
+	defaultSecurityContextGID   = 1000
+	defaultSecurityContextFSGID = 2000
```

Comment:
As discussed offline, would be better to make security settings configurable via cdap-cr. When they are unset, default to current behavior of running as root.

Reply:
Fixed by making it configurable.